ci-operator/step-registry/openshift/image-based/upgrade/e2e/openshift-image-based-upgrade-e2e-commands.sh

```diff
@@ -16,23 +15,20 @@ remote_workdir=$(cat ${SHARED_DIR}/remote_workdir)
 instance_ip=$(cat ${SHARED_DIR}/public_address)
 host=$(cat ${SHARED_DIR}/ssh_user)
 ssh_host_ip="$host@$instance_ip"
-PULL_SECRET_FILE="/var/run/pull-secret/.dockerconfigjson"
-PULL_SECRET=$(cat ${PULL_SECRET_FILE})
+PULL_SECRET_FILE=$(cat ${SHARED_DIR}/pull_secret_file)
 
 TARGET_VM_NAME=$(cat ${SHARED_DIR}/target_vm_name)
 target_kubeconfig=${remote_workdir}/ib-orchestrate-vm/bip-orchestrate-vm/workdir-${TARGET_VM_NAME}/auth/kubeconfig
 remote_artifacts_dir=${remote_workdir}/artifacts
 
 cat <<EOF > ${SHARED_DIR}/e2e_test.sh
 #!/bin/bash
-set -xeuo pipefail
+set -euo pipefail
 
 export KUBECONFIG='${target_kubeconfig}'
-export PULL_SECRET='${PULL_SECRET}'
+export PULL_SECRET=\$(<${PULL_SECRET_FILE})
```

Comment:
I'm confused here. This will try to read the content from the file path, but `PULL_SECRET_FILE` is not a file path anymore but it includes the content of the file because of the 'cat', right?

Reply:
The contents of the file is now the path on the ec2 instance to the `.pull-secret.json`. The value of the pull secret is no longer being passed in these steps but rather read from the file on the ec2 instance. 

The command that would be run on the ec2 instance in this case would look something like `export PULL_SECRET=$(</usr/ec2-user/workdir/.pull-secret.json)` which forward the pull secret into the variable without printing it